### DISCUSSION TITLE
Welcome to Cruiser Discussions!

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DolomitesTom** [comment]
I was re-infected at the age of 45.... from motorcycling. There is no cure and no antidote for it... yet. fortunately. As a Kurviger 1pro and Kurviger 2 user, it was of course clear that I also Cruise (not in the literal sense, it's not my riding style). Let's see what develops out of it. The loss of the Kurviger routing engine is not easy to cope with at first. But there are other much-loved functions of the navigation app available.

**Comment by devemux86** [comment]
@DolomitesTom thanks!

Cruiser development continues at full speed, some new features have already been implemented.
Like a built-in [Download Manager](https://groups.google.com/g/cruiser-users/c/J69JJIoKn-M), new [Biker theme](https://groups.google.com/g/cruiser-users/c/nEx6AyNTDP8) and more surprises are planned for the future!

And you can expect a new [Route overview](https://groups.google.com/g/cruiser-users/c/jO3HUBu7sPU) mode in navigation in the next version.
Edit: already available.

<img src="https://user-images.githubusercontent.com/3484020/209465819-b43c1277-3a8b-43c0-a5e5-d195757c978a.jpg" width="50%"/>

**Comment by Ciutnik** [comment]
Hello everyone! As a longtime fan of Kurviger, I couldn't miss this forum sooner or later. In the face of Kurviger's sudden and unexpected death, I envy Cruiser's development...

I know Cruiser can use different routing engines. My dream is that in the future they will be joined by an engine offering so well known from the past, fantastically curved routes – preferably an offline version :)

Best regards and above all, I wish you only safe travels!
Maciek 'Ciutnik'

**Comment by devemux86** [comment]
@Ciutnik thanks!

> I envy Cruiser's development

Next Cruiser version will have new features in the navigation to adapt it to your **driving style**.

> My dream is that in the future they will be joined by an engine offering so well known from the past, fantastically curved routes – preferably an offline version :)

:+1: 

I am always available for collaborations and include anything in Cruiser.

**Reply by Cer-o** [reply]
> Next Cruiser version will have new features in the navigation to adapt it to your **driving style**.

What do you mean exactly?

**Reply by devemux86** [reply]
I will make an announcement with more details soon here in Cruiser Discussions.

Please wait until the current implementation and testing period is finalized. :slightly_smiling_face:

**Comment by WolfgangZ42** [comment]
It wasn't until I was 50 that I got my motorcycle license. Since then I have been on the road about 500,000 km with the bike. To compensate for the lead of my friends who had started earlier. 
My navigation: the first 12 years with the map, then 7 years with TomTom Urban Rider and now for more than 4 years with Kurviger. 
I planned and plan my tours exclusively in the website (kurviger.de). The exported .kurviger file I imported into the Kurviger app. Since the announced end of the Kurviger app, I navigate (currently only by car) with the Cruiser app. 
Here, too, I import the .kurviger file and drive the exact route without recalculation. If a recalculation should actually become necessary, I prevent a too large influence of BRouter on my route by setting enough shaping points. See [Workflow for Kurviger.pdf ](https://www.file-upload.net/download-15079895/WorkflowforKurviger.pdf.html)
I thank EMUX even in advance for doing an excellent job and know that there will be some more enhancements to the Cruiser app, although it is already almost perfect for me.

**Reply by devemux86** [reply]
@WolfgangZ42 thanks!

Cruiser is far from perfect, there are many big and small features that I can implement and improve the current user experience.
Development always goes on... :slightly_smiling_face:

**Reply by DolomitesTom** [reply]
> Development always goes on... 

This is what we like to hear 🙂

**Comment by devemux86** [comment]
To receive notifications / emails from Cruiser Discussions,
click the "**Watch**" button at the top and select "All Activity".

**Comment by ToToGSARider** [comment]
Hallo zusammen,
so ich hoffe, dass ich das jetzt richtig gemacht habe indem ich mir hier einen Account bei GitHub gemacht habe um hier schreiben und lesen zu können. Ich habe seit 2020 die kurviger 1 und auch die kurviger 2 App im Einsatz. Mein Abo bei der kurviger 2 läuft jetzt am 29.04.2023 aus. Ich bin derzeit etwas hin und her gerissen was ich jetzt machen soll und möchte mich einfach mal umhören bzw. einfach mal mitlesen was beim Cruiser so geht. Da ich das WunderlinQ seit 2020 mein eigen nennen darf, hatte ich damals die perfekte Lösung gesucht und ja auch gefunden. Es bleibt also spannend. Ich bin leider der englischen Sprache nicht ganz so mächtig und nutze deshalb zum lesen den Google Übersetzer und zum schreiben den Deepl, also verzeiht mir schon mal wenn da manchmal komische Dinge stehen. 

Hello all,
so I hope I did this right now by making an account here at GitHub to be able to write and read here. I have been using the kurviger 1 and also the kurviger 2 app since 2020. My subscription to kurviger 2 is now expiring on 04/29/2023. I'm currently a bit torn about what to do now and just want to ask around or just read along what's going on with the Cruiser. Since I can call the WunderlinQ my own since 2020, I was looking for the perfect solution and found it. So it remains exciting. Unfortunately, I'm not quite so powerful English and therefore use the Google Translator to read and the Deepl to write, so forgive me already times if there are sometimes funny things. 

Translated with www.DeepL.com/Translator (free version)

**Reply by devemux86** [reply]
Welcome @ToToGSARider!

Everyone can write in any language they prefer. We can answer in their language (with a translator).

You can try Cruiser, its navigation has advanced a long way and can import (offline) Kurviger routes.

---

Willkommen @ToToGSARider!

Jeder kann in jeder Sprache schreiben, die er bevorzugt. Wir können in ihrer Sprache antworten (mit einem Übersetzer).

Sie können versuchen, Cruiser, seine Navigation hat einen langen Weg fortgeschritten und kann (offline) Kurviger Routen importieren.

**Comment by devemux86** [comment]
I found an old image of Cruiser, you can see how it looked several years ago. :slightly_smiling_face: 

<img src="https://github.com/devemux86/cruiser/assets/3484020/60162fbe-8f01-4cf8-8d44-8b1092ddbe82" width="50%"/>

**Reply by Ciutnik** [reply]
From the outside, a consistent classic, and not much has changed, but under the hood... it's a completely new quality and power 💪
Thanks a lot @devemux86 and Everyone who cares ❤️

**Comment by C6competition** [comment]
Hi,

I like to give my background here, to make understandable, why I have one or the other wishes to Navigation Apps.
I'm a motorbiker since I'm a kind (about 12 years of age) and could not wait to get 18 and own a driving license. From then on,
we drove (My friends & me) everywhere in Europe. Our navigation tool: Harro Elefantenboy, with list of aims under the plastic cover on top!
We got everywhere, but I agree, sometimes we had to stop and look at the map!
Since more than 20 years I now use "modern" navigation equipment. Starting with Garmin hardware, across various other brands of apps on a smart phone. Now I ended at Kurviger, because of their nice PC - Tool. But there are a lot of things to develop in their phone app. Going through their forum, I found the start of the  Cruiser app, called K2. **THAT'S** what I was looking for. Okay, the configuratio structure is not the way, I would design it (I'm used to project and realise MMI's (Maschine Man Interfaces) in my profession), but You can get used to it.
But I have some more ideas for the Cruiser app, I like to bring forward here.

**Reply by devemux86** [reply]
@C6competition Welcome!

Cruiser exists on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser) from 2013.

K1 + K2 apps started many years later, based on Cruiser code.
